extensions/items/standalone.mjs

```diff
@@ -950,6 +967,42 @@ class Standalone {
     }
   }
 
+  /**
+   * Count the total number of items in the given list of items, that is
+   * owned by the player.
+   * @param {Object} kwargs - kwargs.items is an array of string
+   *     representing the items.
+   *   kwargs.haveItems specifies the state to go to if the player have the
+   *     amount of specified items.
+   *   kwargs.noItems specifies the state to go to if the player doesn't have
+   *     the amount of specified items.
+   *   kwargs.amount is the minimum total count to go to "haveItems" state.
+   * @return {String} nextState - The next state.
+   */
+  async s2s_sf_haveMultiItems(srcExt, playerID, kwargs, sfInfo) {
+    let amount = kwargs.amount;
+    let items = kwargs.items;
+
+    if (!Number.isInteger(amount)) amount = 1;
```

Comment:
delete it if above comment resolved

Reply:
if kwargs.amount is `true`, then the result could be incorrect. I've changed it to use Number.isInteger above.